### PR TITLE
add temp schedule

### PR DESCRIPTION
**Pull request by pearlielin**
(Thanks for sending a pull request! Please fill in the following content to let us know better about this change.)

## Types of changes
Please put an `x` in the box that applies

- [ ] **Bugfix**
- [ ] **New feature**
- [ ] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [ ] **Documentation Update**
- [ ] **Other (please describe)**

## Description
**Describe what the change is**
You don't need to explain HOW you change the code. Your code, including comments in the code should explicitly
self-describe enough about how you change the code.

## Related Issue
If applicable, refernce to the issue related to this pull request.

## More Information
**Screenshots**

**Additional context**
Add any other context about the problem here. You may also want to refer
to [how to wirte the perfect pull request](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)

### CODE REVIEW

File: src/pycontw2016/settings/base.py [hunk 1]
```diff
@@ -318,6 +318,6 @@ def node_bin(name):
     (datetime.date(2020, 9, 6), _('Day 2')),
 ])
 
-SCHEDULE_REDIRECT_URL = None
+SCHEDULE_REDIRECT_URL = "https://docs.google.com/spreadsheets/d/1efjxIesuNWaXBPwnaM8XPNpVF9Fo6dStGDjXVNG8AVo"
```
Comment (shunghsiyu) [inline src/pycontw2016/settings/base.py:321]: 這個 2020 年 schedule 網址設定好像不用寫在 `base.py` （因為會被各年份的設定檔 import ），寫在 `src/pycontw2016/settings/production/pycontw2020.py` 應該就好了？
（類似 `pycontw2019.py` 設定檔的作法）

**Comment by josix** [comment]
目前 CI 因為 django test client 沒有辦法存取外部網站 (見 [1]，_The test client is not capable of retrieving Web pages that are not powered by your Django project_. ）導致測試時 server 會對 redirect 後的 request 做處理進而造成 [test_content_pages_links](https://github.com/pycontw/pycon.tw/blob/master/src/core/tests/test_views.py#L139) 沒有通過，這邊 test_content_pages_links 需要再做修正加入 assertRedirect 的判斷 [2]，我這邊會再發另外一個 PR ，也許要等待該 PR merge 後這個 PR 可以再繼續進行，謝謝🙏

[1] https://docs.djangoproject.com/en/3.0/topics/testing/tools/#overview-and-a-quick-example
[2] https://docs.djangoproject.com/en/3.0/topics/testing/tools/#django.test.SimpleTestCase.assertRedirects

Related issues:
- https://stackoverflow.com/questions/27390796/django-testing-redirects-to-external-pages
- https://stackoverflow.com/questions/39707524/testing-external-urls-in-django

File: src/pycontw2016/settings/base.py (hunk 1 above)
Comment (yychen) [reply src/pycontw2016/settings/base.py:321]: 對，應該要寫在 pycontw2020.py 裡面，再請 @pearlielin 改一下。

File: src/events/views.py [hunk 2]
```diff
@@ -99,14 +99,16 @@ class ScheduleView(TemplateView):
...
+        # try:
+        #     self.schedule = Schedule.objects.latest()
+        # except Schedule.DoesNotExist:
+        #     redirect_url = settings.SCHEDULE_REDIRECT_URL
+        #     if not redirect_url:
+        #         return HttpResponseNotFound()
+        #     return HttpResponseRedirect(redirect_url)
+        # return super().get(request, *args, **kwargs)
+        redirect_url = settings.SCHEDULE_REDIRECT_URL
+        return HttpResponseRedirect(redirect_url)
```
Comment (yychen) [inline src/events/views.py:111]: 這邊我建議就是維持原狀就好了。目前資料庫裡不會有 Schedule 的資料，所以他會直接到 105 行這裡面。等到到時候資料庫裏面有資料了之後就會直接跳到正確的時間表頁面了。

**Pull request merged**